api/envoy/config/filter/http/grpc_http1_reverse_bridge/v2alpha1/config.proto

```diff
@@ -24,3 +24,14 @@ message FilterConfig {
   // simple binary encoded protobuf.
   bool withhold_grpc_frames = 2;
 }
+
+// gRPC reverse bridge filter configuration per virtualhost/route/weighted-cluster level.
+message FilterConfigPerRoute {
+  // It disables gRPC reverse bridge filter for this particular vhost or route.
+  // If disabled is specified in multiple per-filter-configs, the most specific one will be used.
+  // The order is:
+  // - the routeEntry() (for config that's applied on weighted clusters)
+  // - the route
+  // - and finally from the virtual host object (routeEntry()->virtualhost()).
```

Comment:
I think we can remove the section about how per-filter configs work since this behavior is common among all HTTP filters that support per-route configuration. If you feel it needs to be more explicitly documented, let's find a more generic place to describe the behavior.

grammar nit: Let's start the first sentence with "If true, disables the gRPC reverse bridge" (and so on).

Reply:
Fixed in https://github.com/envoyproxy/envoy/pull/8645/commits/0edac0420944a19891d43804acbe6fd3f0edf654